### PR TITLE
Create Tasklist

### PR DESCRIPTION
**Pull request by benedict733**
Update from Task to Tasklist

AddTaskCommand no longer refreshes single Task, now adds Task to TaskList

ClearTaskCommand clears all Tasks

### CODE REVIEW

**Review by evanpy — COMMENTED** [review]
Good job! Few minor fixes and its good to go! 👍🏻

File: src/main/java/seedu/address/logic/parser/AddCommandParser.java [hunk 1]
```diff
@@ -64,10 +64,10 @@ public AddCommand parse(String args) throws ParseException {
         Address address = ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get());
         Remark remark = new Remark(""); // add command does not allow adding remarks straight away
         Set<Tag> tagList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG));
-        Task task = new Task("");
+        TaskList tasks = new TaskList();
 
         Person person = new Person(name, gender, phone, email, company, industry, occupation, jobTitle, address,
-                remark, tagList, task);
+                remark, tagList, tasks);
```
[the remark this answers is not in the document]
Comment (benedict733) [reply src/main/java/seedu/address/logic/parser/AddCommandParser.java:70]: no it is not compulsory, have included a comment for variable tasks for this (similar to remark)

File: src/test/java/seedu/address/logic/commands/AddTaskCommandTest.java [hunk 2]
```diff
@@ -37,7 +37,7 @@ public void execute_addTaskUnfilteredList_success() {
         Person firstPerson = model.getFilteredPersonList().get(INDEX_FIRST_PERSON.getZeroBased());
         Person editedPerson = new PersonBuilder(firstPerson).withTask(TASK_STUB).build();
 
-        AddTaskCommand addTaskCommand = new AddTaskCommand(INDEX_FIRST_PERSON, new Task(editedPerson.getTask().value));
+        AddTaskCommand addTaskCommand = new AddTaskCommand(INDEX_FIRST_PERSON, new Task(TASK_STUB));
```
[the remark this answers is not in the document]
Comment (benedict733) [reply src/test/java/seedu/address/logic/commands/AddTaskCommandTest.java:40]: the file here is a unit test file for AddTaskCommand, can i check if you are mistaken here?

File: src/main/java/seedu/address/logic/commands/ClearTaskCommand.java [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+                personToEdit.getTags(), EMPTY_TASKLIST, personToEdit.getStatus());
+
+        model.setPerson(personToEdit, editedPerson);
+        model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
+
+        return new CommandResult(generateSuccessMessage(editedPerson));
+    }
+
+    /**
+     * Generates a command execution success message based on whether the remark is added to or removed from
```
[the remark this answers is not in the document]
Comment (benedict733) [reply src/main/java/seedu/address/logic/commands/ClearTaskCommand.java]: updated in next commit

File: src/main/java/seedu/address/logic/parser/ClearCommandParser.java [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.ClearTaskCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new DeleteCommand object
+ */
+public class ClearCommandParser implements Parser<ClearTaskCommand> {
```
[the remark this answers is not in the document]
Comment (benedict733) [reply src/main/java/seedu/address/logic/parser/ClearCommandParser.java]: have renamed in next commit

File: src/test/java/seedu/address/logic/commands/AddTaskCommandTest.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (evanpy) [reply src/test/java/seedu/address/logic/commands/AddTaskCommandTest.java:40]: Yeap, I am! Sorry!

**Pull request merged**